sparse_strips/vello_sparse_tests/tests/renderer.rs

```diff
@@ -217,6 +217,21 @@ impl Renderer for Scene {
     }
 
     fn render_to_pixmap(&self, pixmap: &mut Pixmap, _: RenderMode) {
```

Comment:
Whilst we're here, can we have a comment that this method implementation is awful and should never be used outside of tests? (to be clear, this is as true before this PR as after)